<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UCSManager</class>
 <widget class="QMainWindow" name="UCSManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>1002</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UCSManager V1.2 by Pump It Up Gallery</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>PIU Login</string>
    </property>
    <widget class="QLineEdit" name="pwBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
     <property name="placeholderText">
      <string>PW</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="idBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>ID</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="loginStatus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Login Status : Not Logged In</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="isSaveAccount">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto Login</string>
     </property>
    </widget>
    <zorder>idBox</zorder>
    <zorder>loginButton</zorder>
    <zorder>loginStatus</zorder>
    <zorder>pwBox</zorder>
    <zorder>isSaveAccount</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>481</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>UCS List</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>2.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>3.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>4.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>5.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>6.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>7.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>8.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>9.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>10.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="registerButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Build UCS Pack</string>
     </property>
    </widget>
    <widget class="QLabel" name="songInfo1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Not Registered.</string>
     </property>
    </widget>
    <widget class="QLabel" name="songInfo2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Not Registered.</string>
     </property>
    </widget>
    <widget class="QLabel" name="songInfo3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Not Registered.</string>
     </property>
    </widget>
    <widget class="QLabel" name="songInfo4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Not Registered.</string>
     </property>
    </widget>
    <widget class="QLabel" name="songInfo5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Not Registered.</string>
     </property>
    </widget>
    <widget class="QLabel" name="songInfo6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Not Registered.</string>
     </property>
    </widget>
    <widget class="QLabel" name="songInfo9">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>270</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Not Registered.</string>
     </property>
    </widget>
    <widget class="QLabel" name="songInfo7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Not Registered.</string>
     </property>
    </widget>
    <widget class="QLabel" name="songInfo8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Not Registered.</string>
     </property>
    </widget>
    <widget class="QLabel" name="songInfo10">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>300</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Not Registered.</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>481</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Delete UCS</string>
    </property>
    <widget class="QComboBox" name="deleteListBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear All UCS</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>481</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Find UCS</string>
    </property>
    <widget class="QLineEdit" name="stepArtistBox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Step Artist</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="songTitleBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Song Title</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="stepLevelBox">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Level</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resultAddButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QComboBox" name="searchResultBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Add UCS</string>
    </property>
    <widget class="QLineEdit" name="ucsListBox">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>30</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>UCSNo, ㆍㆍㆍ , UCSNo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ucsListAddButton">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>80</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>920</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Log</string>
    </property>
    <widget class="QLabel" name="logText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Welcome!</string>
     </property>
    </widget>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>231</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>   Made by qwertycvb</string>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>idBox</tabstop>
  <tabstop>pwBox</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>ucsListBox</tabstop>
  <tabstop>ucsListAddButton</tabstop>
  <tabstop>songTitleBox</tabstop>
  <tabstop>stepArtistBox</tabstop>
  <tabstop>stepLevelBox</tabstop>
  <tabstop>searchResultBox</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>resultAddButton</tabstop>
  <tabstop>deleteListBox</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>registerButton</tabstop>
  <tabstop>commandLinkButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
